tests/test_gateways.py

```diff
@@ -117,3 +117,15 @@ def test_gateway(self, logger):
             fake.send_sms(device=Mock(number=PhoneNumber.from_string('+123')), token=code)
             logger.info.assert_called_with(
                 'Fake SMS to %s: "Your token is: %s"', '+123', code)
+
+
+class QueryableFakeGatewayTest(TestCase):
```

Comment:
More tests please. E.g. do `call_tokens` and `sms_tokens` get reset properly after each testcase?